lib/Validator/LengthValidator.php

```diff
@@ -49,6 +49,15 @@ public function validate(string $password): void {
 			);
 			throw new HintException($message, $message_t);
 		}
+
+		$maxLength = $this->config->getMaxLength();
+		if(strlen($password) > $maxLength) {
+			$message = 'Password needs to be at most ' . $maxLength . ' characters long';
+			$message_t = $this->l->t(
+				'Password needs to be at most %s characters long', [$maxLength]
+			);
```

Comment:
```suggestion
			$message_t = $this->l->n(
				'Password needs to be at most %n character long',
				'Password needs to be at most %n characters long',
				$maxLength
			);
```

need plurals (as there are languages which dont go by 1 or multiple, but by (number ends with 1, with 2, with 3-5, ...)